src/amg/azext_amg/tests/latest/test_amg_scenario.py

```diff
@@ -440,6 +440,89 @@ def test_amg_backup_restore(self, resource_group):
                 self.check("length([?uid == '{dashboardUid}'])", 1)
             ])
 
+    @ResourceGroupPreparer(name_prefix='cli_test_amg')
+    def test_amg_private_endpoint(self, resource_group):
+
+        self.kwargs.update({
+            'name': 'clitestamgmpe',
+            'location': 'westcentralus',
+            'monitor_name': 'clitestamgmonitor',
+            'mpe_name': 'clitestmpe',
+            'pe_name': 'clitestpe',
+            'vnet_name': 'clitestvnet',
+            'subnet_name': 'clitestsubnet',
+            'conn_name': 'clitestconn',
+            'sub': self.get_subscription_id(),
+        })
```

Comment:
consider using random names to avoid collisions: https://github.com/Azure/azure-cli/blob/master/doc/authoring_tests.md#sample-6-random-name-and-name-mapping